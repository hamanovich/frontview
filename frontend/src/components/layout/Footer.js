import React from 'react';
import styled from 'styled-components';

const Foot = styled.footer`
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 60px;
  padding: 10px 0;
  background-color: #1a242f;
  text-align: center;
  color: #fff;
`;

const Footer = () => (
  <Foot>
    &copy; Front View <br /> 2017-2018
  </Foot>
);

export default Footer;
