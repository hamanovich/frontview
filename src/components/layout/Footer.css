.footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 60px;
  padding: 10px 0;
  background-color: #1a242f;
}
