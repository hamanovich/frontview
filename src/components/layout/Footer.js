import React from 'react';

import Grid from 'react-bootstrap/lib/Grid';

const Footer = () => (
  <footer className="footer text-center">
    <Grid>
      <p className="text-muted">&copy; Front View <br /> 2017</p>
    </Grid>
  </footer>
);

export default Footer;