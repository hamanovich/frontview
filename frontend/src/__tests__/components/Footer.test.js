import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../components/layout/Footer';

describe('<Footer/>', () => {
  it('renders <Footer /> component', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
