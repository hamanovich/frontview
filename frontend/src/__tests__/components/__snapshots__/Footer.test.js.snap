// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders <Footer /> component 1`] = `
<styled.footer>
  Â© Front View 
  <br />
  <Badge
    bsClass="badge"
    pullRight={false}
  >
    v1.0.0
  </Badge>
  <br />
  <a
    href="mailto:hamanovich@gmail.com"
  >
    hamanovich@gmail.com
  </a>
   
  <br />
  2017-
  2019
</styled.footer>
`;
